cmd/mt-whisper-importer-reader/conversion.go

```diff
@@ -51,17 +52,17 @@ func (c *conversion) getPoints(retIdx int, spp, nop uint32) map[string][]whisper
 	smallestArchiveIdx, largestArchiveIdx := c.findSmallestLargestArchive(spp, nop)
 	rawRes := c.archives[0].SecondsPerPoint
 
-	adjustedPoints := make(map[string]map[uint32]float64)
-	if retIdx > 0 && c.method == "avg" || c.method == "sum" {
-		adjustedPoints["cnt"] = make(map[uint32]float64)
-		adjustedPoints["sum"] = make(map[uint32]float64)
+	adjustedPoints := make(map[schema.Method]map[uint32]float64)
+	if retIdx > 0 && c.method == schema.Avg || c.method == schema.Sum {
+		adjustedPoints[schema.Cnt] = make(map[uint32]float64)
```

Comment:
why do we keep a count aggregation if all the user wants is 'sum'?

Reply:
I think that was because that we can also serve average queries

Reply:
we should stop doing that.  The "count" value we are storing will be wrong any time the original metrics stream had null points.  eg, if the raw interval is 1min and their is a 10min SUM rollup, we cant assume that the sum is the result of 10 points.  It could be the result of anywhere from 0-10 points.

But maybe we should address this problem in a followup PR.